### PR TITLE
TinyFD API + various bug fixes

### PR DESCRIPTION
**Pull request by Wyvest**
## Description
- Updates PGT to 0.2.9
- Adds new TinyFD API (we need this for EvergreenHUD)
- Outright force disable fast render and anti-aliasing (these features aren't useful anyways and break OneConfig a lot)
- Remove blank names from Vigilance mod compat detection

## Related Issue(s)
N/A

## Checklist
- [x] I made a clear description of what was changed
- [x] I stated why these changes were necessary
- [x] I updated documentation or said what needs to be updated
- [x] I made sure these changes are backwards compatible

### CODE REVIEW

File: src/main/java/cc/polyfrost/oneconfig/renderer/TinyFD.java [hunk 1]
```diff
@@ -1,4 +1,70 @@
...
-}
+    String QUESTION_ICON = "question";
+    String ERROR_ICON = "error";
+    String WARNING_ICON = "warning";
+    String INFO_ICON = "info";
+
+    String OK_DIALOG = "ok";
+    String OK_CANCEL_DIALOG = "okcancel";
+    String YES_NO_DIALOG = "yesno";
+    String YES_NO_CANCEL_DIALOG = "yesnocancel";
```
Comment (pauliesnug) [inline src/main/java/cc/polyfrost/oneconfig/renderer/TinyFD.java:18]: i18n when

File: versions/src/main/java/cc/polyfrost/oneconfig/internal/plugin/hooks/VigilantHook.java [hunk 2]
```diff
@@ -41,6 +42,7 @@ public class VigilantHook {
     public static VigilanceConfig returnNewConfig(Vigilant vigilant, File file) {
         if (vigilant != null && Platform.getInstance().isCallingFromMinecraftThread()) {
             String name = !vigilant.getGuiTitle().equals("Settings") ? vigilant.getGuiTitle() : !Platform.getLoaderPlatform().hasActiveModContainer() ? "Unknown" : Platform.getLoaderPlatform().getActiveModContainer().name;
+            if (StringsKt.isBlank(name)) return null;
```
Comment (pauliesnug) [inline versions/src/main/java/cc/polyfrost/oneconfig/internal/plugin/hooks/VigilantHook.java:45]: why?

**Review by pauliesnug — APPROVED** [review]
This pull request by Wyvest is magnificent!

File: src/main/java/cc/polyfrost/oneconfig/renderer/TinyFD.java (hunk 1 above)
Comment (Wyvest) [reply src/main/java/cc/polyfrost/oneconfig/renderer/TinyFD.java:18]: this is just TinyFD stuff iirc

File: versions/src/main/java/cc/polyfrost/oneconfig/internal/plugin/hooks/VigilantHook.java (hunk 2 above)
Comment (Wyvest) [reply versions/src/main/java/cc/polyfrost/oneconfig/internal/plugin/hooks/VigilantHook.java:45]: image

File: src/main/java/cc/polyfrost/oneconfig/renderer/TinyFD.java (hunk 1 above)
Comment (pauliesnug) [reply src/main/java/cc/polyfrost/oneconfig/renderer/TinyFD.java:18]: > this is just TinyFD stuff iirc

ah okay, does tinyfd expose these constants?

Comment (Wyvest) [reply src/main/java/cc/polyfrost/oneconfig/renderer/TinyFD.java:18]: ye

**Review by xtrm-en — APPROVED** [review]

**Review by nextdayy — COMMENTED** [review]
looks good to me as long as the force disable works

**Pull request merged**